docs/reference/manifest/runtimes.md

```diff
@@ -8,9 +8,6 @@ localization_priority: Normal
 
 Specifies the runtime of your add-in. Child of the [`<Host>`](host.md) element.
 
-> [!NOTE]
-> When running in Office on Windows, your add-in uses the Internet Explorer 11 browser.
-
 In Excel, this element enables the ribbon, task pane, and custom functions to use the same runtime. For more information, see [Configure your Excel add-in to use a shared JavaScript runtime](../../excel/configure-your-add-in-to-use-a-shared-runtime.md).
```

Comment:
@davidchesnut This article has a note to the same effect. Does that need to be updated/removed as well?